controllers/articles.controller.js

```diff
@@ -1,22 +1,33 @@
-const { fetchArticleById, fetchArticles } = require('../models/articles.model')
+const { fetchArticleById, fetchArticles, fetchCommentsByArticleId } = require(
+  '../models/articles.model');
 
 const getArticleById = (request, response, next) => {
 
   const { article_id } = request.params;
 
   fetchArticleById(article_id).then((article) => {
-    response.status(200).send({article});
-  }).catch((err) =>{
+    response.status(200).send({ article });
+  }).catch((err) => {
     next(err);
-  })
-}
+  });
+};
 
 const getArticles = (request, response, next) => {
   fetchArticles().then((articles) => {
-    response.status(200).send({articles});
-  }).catch((err) =>{
+    response.status(200).send({ articles });
+  }).catch((err) => {
     next(err);
-  })
-}
+  });
+};
 
-module.exports = { getArticleById, getArticles };
\ No newline at end of file
+const getCommentsByArticleId = (request, response, next) => {
+  const { article_id } = request.params;
+
+  fetchCommentsByArticleId(article_id).then((comments) => {
+    response.status(200).send({ comments });
+  }).catch((err) => {
+    next(err);
+  });
+};
```

Comment:
How do you know the article id exists? For example in your test you put in article id, but you don't seem to be any check to see if the article id exists before getting the comments.

Comments will only exist on an article_id which exists, and there will either be comments or no current comments on an article_id which exists. 

Think about how you're going to check the article exists _before_ going to get comments.

Everything else in this controller function is good though, along with sending the response back on a key of `comments`